app/src/types.ts

```diff
@@ -0,0 +1,24 @@
+export interface Room {
+  id: string;
+  name: string;
+}
+
+export interface Speaker {
+  id: string;
+  name: string;
+  email: string;
+  avatar: null;
```

Comment:
This fields with null, they can be a different value right? like string perhaps? so what if string | null?

Reply:
ohh, I used the quicktype plugin to generate these, missed that one, sorry :) It's actually an image from strapi I think it's an object with some props like url. I'll update this

Reply:
Fixed :D